npbench/infrastructure/jax_framework.py

```diff
@@ -25,6 +30,66 @@ def copy_func(self) -> Callable:
         for copying the benchmark arguments. """
         return jnp.array
 
+    def impl_files(self, bench: Benchmark):
+        """ Returns the framework's implementation files for a particular
+        benchmark.
+        :param bench: A benchmark.
+        :returns: A list of the benchmark implementation files.
+        """
+
+        parent_folder = pathlib.Path(__file__).parent.absolute()
+        implementations = []
+        for impl_name, impl_postfix in _impl.items():
+            pymod_path = parent_folder.joinpath(
+                "..", "..", "npbench", "benchmarks", bench.info["relative_path"],
+                bench.info["module_name"] + "_" + self.info["postfix"] + "_" + impl_postfix + ".py")
+            implementations.append((pymod_path, impl_name))
+        
+        # appending the default implementation
+        pymod_path = parent_folder.joinpath("..", "..", "npbench", "benchmarks", bench.info["relative_path"],
+                                            bench.info["module_name"] + "_" + self.info["postfix"] + ".py")
+        
+        implementations.append((pymod_path, 'default'))
```

Comment:
Can we have default as the first member in `implementations` list? Should not make a difference but just to ensure something does not break in the future. 

Numpy default implementation is hardcoded to be indexed as the 0th indexed element in the implementations list: [npbench/infrastructure/test.py#L64](https://github.com/spcl/npbench/blob/0bc108b70579d4691ec4bd3555b16431bebb881a/npbench/infrastructure/test.py#L64)